docs/pfsense_installerconfig.md

```diff
@@ -1,11 +1,87 @@
-Unattanded installation PFSense
+Unattended installation PFSense
 ===============================
 
+This document describes the process of preparing the automatic installer pFsense
+on a USB stick. The whole procedure was tested on Ubuntu with the kernel
+5.15.0-58-generic, the download and installation procedure of the kernel is also
+described below.
+
+Kernel installation
+-------------------
+
+1. Download the required packages by running the following command:
+
+   ```bash
+   sudo apt install linux-headers-5.15.0-58-generic linux-source-5.15.0
```

Comment:
Please try to use `uname` to make the procedure generic

Reply:
@Pokisiekk `uname -r` results in `5.15.0-58-generic`. So... 
`sudo apt install linux-headers-$(uname -r) linux-source` should make it generic. `linux-source` package should download the necessary version already.

Try to use uname in commands wherever applicable

Reply:
In many cases we refer to  version of linux-source(5.15.0), which is why I used the command `$(uname -r | cut -d- -f1)` = 5.15.0. Now documentation should be really generic.